vue/ci.yml

```
@@ -0,0 +1,22 @@
+# Example pipeline definition, for react apps.
+#
+# Note: If you have very long and/or complex commands,
+# consider writing a script and execute it from here.
+
+prepare:
+  # Put your commands here.
+  steps:
+  - name: "Install dependencies"
+    command: "yarn install"
+  - name: "Build"
+    command: "yarn build"
+
+test:
+  # Put your commands here.
+  steps: []
+
+run:
+  # Put your commands here.
+  steps:
+  - name: "Run"
+    command: "yarn preview"
```

```suggestion
    command: "yarn preview --host --port 3000"
```
Please test if your app actually runs on Codesphere with your CI pipeline